client/state/sites/selectors.js

```diff
@@ -142,6 +143,26 @@ export function getSitePlan( state, siteId ) {
 		return null;
 	}
 
+	if ( get( site.plan, 'expired', false ) ) {
+		if ( site.jetpack ) {
+			return {
+				product_id: 2002,
+				product_slug: 'jetpack_free',
+				product_name_short: 'Free',
+				free_trial: false,
+				expired: false
+			};
+		}
+
+		return {
```

Comment:
I don't understand why `getSitePlan` should return this object if the plan is expired. Shouldn't it simply return the currently expired plan with `expired` set to `true`? What's the use case of this? :)